content/en/news/releases/1.7.x/announcing-1.7.5/index.md

```diff
@@ -25,6 +25,6 @@ This release contains bug fixes to improve robustness. This release note describ
 
 - **Fixed** multicluster `EnvoyFilter` to have valid configuration following the underlying changes in Envoy’s API. ([Issue #27909](https://github.com/istio/istio/issues/27909))
 
-- **Fixed** an issue causing a short spike in errors during in place upgrades from Istio 1.6 to 1.7. Previously, the TLS version would be upgraded automatically from `TLSv2` to `TLSv3`. This caused a spike in errors if there was a mix of Istio 1.6 and Istio 1.7 proxies running. The `PILOT_ENABLE_TLS_XDS_DYNAMIC_TYPES`=false environment variable was added to disable automatic TLS version updates. This is enabled by default but should be disabled for upgrades from Istio 1.6 to Istio 1.7. ([Issue #28120](https://github.com/istio/istio/issues/28120))
+- **Fixed** an issue causing a short spike in errors during in place upgrades from Istio 1.6 to 1.7. Previously, the xDS version would be upgraded automatically from xDS v2 to xDS v3. This caused downtime with upgrades from Istio 1.6 to Istio 1.7. This has been fixed so that these upgrades no longer cause downtime. Note that, as a trade-off, upgrading from Istio 1.7.x to Istio 1.7.5 still causes downtime in any existing 1.6 proxies; if you are in this scenario you may set the `PILOT_ENABLE_TLS_XDS_DYNAMIC_TYPES` environment variable to false in Istiod to retain the previous behavior. ([Issue #28120](https://github.com/istio/istio/issues/28120))
```

Comment:
```suggestion
- **Fixed** an issue causing a short spike in errors during in place upgrades from Istio 1.6 to 1.7. Previously, the xDS version would be upgraded automatically from xDS v2 to xDS v3. This caused downtime with upgrades from Istio 1.6 to Istio 1.7. This has been fixed so that these upgrades no longer cause downtime. Note that, as a trade off, upgrading from Istio 1.7.x to Istio 1.7.5 still causes downtime in any existing 1.6 proxies; if you are in this scenario you may set the `PILOT_ENABLE_TLS_XDS_DYNAMIC_TYPES` environment variable to false in Istiod to retain the previous behavior. ([Issue #28120](https://github.com/istio/istio/issues/28120))
```